src/utils/sprites/positioning.py

```diff
@@ -0,0 +1,104 @@
+from pyglet.math import Vec2
+import arcade
+
+from src.gui.window_data import WindowData
+from src.utils.pathing.grid_utils import Node
+
+
+def grid_offset(
+    x: int,
+    y: int,
+    window_scale: tuple[float, float],
+    tile_base_dims: tuple[int, int],
+    scale_factor: float,
+    width: int,
+    height: int,
+) -> Vec2:
+    grid_scale = 0.75
+    sx, sy = window_scale
+    constant_scale = grid_scale * tile_base_dims[0] * scale_factor
+    return Vec2(
+        (x - y) * sx * 13,
+        (x + y) * sy * 5,
+    ) * constant_scale + Vec2(width / 2, 7 * height / 8)

```

Comment:
Just a thought on these changes, I think this is the right direction (i.e. moving the grid position to its own module)  but looking at it a it currently is it occurs to me that the implementation of this function was very much coupled to the class we originally implemented it in. 

I think we can probably collapse the args here into a much smaller set of paramteters, for example the parts that all get multiplied together (`grid_scale`, `tile_base_dims`, `scale_factor`) can probably be done in the calling scope and passed as a single scaling parameter.